### PR TITLE
cp: Only use session when -c is passed

### PR DESCRIPTION
**Pull request by vadmeste**

### CODE REVIEW

File: cmd/progress-bar.go [hunk 1]
```diff
@@ -73,9 +73,9 @@ func newProgressBar(total int64) *progressBar {
 	}
 
 	// Start the progress bar.
-	if bar.Total > 0 {
-		bar.Start()
-	}
+	// if bar.Total > 0 {
+	bar.Start()
+	// }
```
Comment (balamurugana) [inline cmd/progress-bar.go]: May be these comments can be removed.

**Comment by sinhaashish** [comment]
need to add the `-c ` in the examples as well as `minio-client-complete-guide.md` for users to know about this functionality.

File: cmd/progress-bar.go (hunk 1 above)
Comment (kannappanr) [reply cmd/progress-bar.go]: done

**Comment by harshavardhana** [comment]
PTAL @sinhaashish @balamurugana

**Pull request merged**